HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] Check isAvailable when using TargetEXE Setting

Found this small bug while working on another PR.
If using an alternative EXE for hyperplay games or sideloaded games it won't check correctly if it is available or not, it will always check the original exe.

### To Test
- Delete or rename the game EXE.
- Going to the game page it will show as game not available.
- Go to the game settings and select an alternative EXE. For instance, the renamed one.
- On `main` and stable it will keep saying that the game is not available.
- On this branch it should show as game available and then launch the alternative EXE.

Related QASE Suites:
- https://app.qase.io/project/HC?suite=72&previewMode=modal&case=360
- https://app.qase.io/project/HC?suite=72&previewMode=modal&case=361

---

Use the following Checklist if you have changed something on the Backend or Frontend:
